openshift/kustomize/tekton/base/triggers/dev/ingress.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: git-webhook
+  name: git-webhook-dev
```

Comment:
curious about the use of the word/token `dev` here.  Are we tying the use of this too closely to the `dev` branch or environment?

Reply:
It's a simple way to know which branch is using this webhook.
Originally I wanted to use a single webhook from Github, but I couldn't figure out how to make it trigger different pipelines.  I've always run into some kind of limitation either in Tekton or in my knowledge.